polaris-react/src/components/Box/Box.stories.tsx

```diff
@@ -33,56 +33,40 @@ export function WithBorders() {
       <Box
         background="bg"
         padding="400"
-        borderWidth="2"
+        borderWidth="050"
         borderColor="border-info"
       >
         <Text as="p">2px solid blue</Text>
       </Box>
       <Box
         background="bg"
         padding="400"
-        borderWidth="3"
-        borderColor="border-success"
-      >
-        <Text as="p">3px solid green</Text>
-      </Box>
-      <Box
-        background="bg"
-        padding="400"
-        borderWidth="4"
+        borderWidth="100"
         borderColor="border-caution"
       >
         <Text as="p">4px solid yellow</Text>
       </Box>
       <Box
         background="bg"
         padding="400"
-        borderWidth="5"
+        borderBlockStartWidth="100"
         borderColor="border-critical"
       >
-        <Text as="p">5px solid red</Text>
+        <Text as="p">border-block-start: 4px solid red</Text>
       </Box>
       <Box
         background="bg"
         padding="400"
-        borderBlockStartWidth="5"
+        borderWidth="100"
+        borderBlockStartWidth="025"
         borderColor="border-critical"
       >
-        <Text as="p">border-block-start: 5px solid red</Text>
-      </Box>
-      <Box
-        background="bg"
-        padding="400"
-        borderWidth="5"
-        borderBlockStartWidth="1"
-        borderColor="border-critical"
-      >
-        <Text as="p">border-width: 5px solid red</Text>
```

Comment:
This diff is a bit more complex than just a 1:1 replacement but I basically removed the `<Box borderWidth="5">` because it's redundant with the `<Box borderWidth="100">` example since `--p-border-width-5` is replaced with `--p-border-width-100` example. I replaced it with the `<Box borderWidth="5" borderBlockStartWidth="1">` example instead.